support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouSupportReminder.jsx

```diff
@@ -80,17 +87,35 @@ const getDefaultLabel = (date: Date, monthsUntilDate: number, now: Date) => {
   const year =
     now.getFullYear() === date.getFullYear() ? '' : ` ${date.getFullYear()}`;
 
-  return `in ${monthsUntilDate} months (${month}${year})`;
+  return `in ${monthsUntilDate} months only (${month}${year})`;
+};
+
+const getRecurringReminderChoice = (): ReminderChoice => {
+  const now = new Date();
+  const startMonth = now.getMonth() + (now.getDate() >= 20 ? 1 : 0); // TODO - is this right?
```

Comment:
I think it'll be `2 : 1`

Reply:
https://github.com/guardian/support-dotcom-components/blob/main/src/lib/reminderFields.ts#L8

Reply:
that's the equivalent in SDC

Reply:
oops, you're quite right!